packages/control-property-editor/src/slice.ts

```diff
@@ -446,6 +457,48 @@ const slice = createSlice<SliceState, SliceCaseReducers<SliceState>, string>({
                 }
             )
             .addMatcher(
+                showInfoCenterMessage.match,
+                (state: SliceState, action: ReturnType<typeof showInfoCenterMessage>): void => {
+                    state.infoCenterMessages.unshift(action.payload);
+                }
+            )
+            .addMatcher(
+                clearInfoCenterMessage.match,
+                (state: SliceState, action: ReturnType<typeof clearInfoCenterMessage>): void => {
+                    state.infoCenterMessages = state.infoCenterMessages.filter((_, index) => index !== action.payload);
```

Comment:
All of these need to work with unique ids a not to rely on the order of elements in the array.